projects/wireshark/build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# Wireshark build.sh script inspired from projects/gnutls/build.sh by Alex Gaynor.
+
+FUZZ_DISSECTORS="ip \
+  udp"
+
+export WIRESHARK_LIBS_PATH="$SRC/libs"
+mkdir -p "$WIRESHARK_LIBS_PATH"
+
+# compile static version of libs
+# XXX, with static wireshark linking each fuzzer binary is ~240 MB (just libwireshark.a is 423 MBs).
+# XXX, wireshark build infrastucture is not ready for including plugins into static binary.
+CONFOPTS="--disable-shared --enable-static --without-plugins"
+
+# disable optional dependencies
+CONFOPTS="$CONFOPTS --without-pcap --without-ssl --without-gnutls"
+
+# need only libs, disable programs
+CONFOPTS="$CONFOPTS --disable-wireshark --disable-tshark --disable-sharkd \
+             --disable-dumpcap --disable-capinfos --disable-captype --disable-randpkt --disable-dftest \
+             --disable-editcap --disable-mergecap --disable-reordercap --disable-text2pcap \
+             --without-extcap \
+         "
+
+./autogen.sh
+./configure --prefix="$WIRESHARK_LIBS_PATH" $CONFOPTS --disable-warnings-as-errors
+
+make "-j$(nproc)"
+make install
+
+WIRESHARK_FUZZERS_COMMON_FLAGS="-lFuzzingEngine /usr/local/lib/libc++.a \
```

Comment:
It seems to be compile without it, based on ffmpeg/, maybe it can be also removed there?